test/factories.rb

```diff
@@ -63,14 +63,14 @@ def create_new_stage(options = {})
   #     :name => random_string,
   #     :primary => 0
   #   }.update(options)
-  #   
+  #
   #   r = Role.new
   #   r.name = options[:name]
   #   r.stage = options[:stage]
   #   r.host = options[:host]
   #   r.primary = options[:primary]
   #   r.save!
-  #   
+  #
   #   return r
   # end
```

Comment:
Please go ahead and delete commented out code that you encounter, unless there's a good comment explaining why we need to keep it around. You can always fetch the code back from your VCS.